docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,177 @@
+
+* TOC
+{:toc}
+
+## How to use it
+In order to use priority and preemption in Kubernetes 1.8, you should follow these
+steps:
+
+1. Enable Priority and Preemption.
+1. Add one or more PriorityClasses.
+1. Create pods with `PriorityClassName` set to one of the added PriorityClasses.
+
+The following sections provide more information about these steps.
+
+## Enable Priority and Preemption
+Pod priority and preemption is disabled by default in Kubernetes 1.8 as it is an
+__alpha__ feature. It can be enabled by a command-line flag:
+
+```
+--feature-gates=PodPriority=true
+```
+
+Once enabled you can add PriorityClasses and create pods with `PriorityClassName` set.
+If you tried it and decided to disable it, you must remove this command-line flag or
+set it to false and restart API server and Scheduler. Once disabled, the existing
+pods may keep their priority fields, but preemption will be disabled and priority
+fields will be ignored.
+
+**Note:** Alpha features should not be used in production systems! Alpha 
+features are more likely to have bugs and future changes to them are not guaranteed to
+be backward compatible.
+
+## PriorityClass
+A PriorityClass object defines a mapping from a PriorityClassName to the integer
+value of the priority. The higher the value, the higher the priority. PriorityClass
+objects can have any 32-bit integer value smaller than or equal to 1 billion. Larger
+numbers are reserved for system use.
+
+PriorityClass also has two optional fields: `globaleDefault` and `description`.
+`globalDefault` indicates that the value of this PriorityClass should be used for
+pods without a `PriorityClassName`. Only one PriorityClass with `globalDefault` 
+set to true can exists in the system. If there is no PriorityClass with `globalDefault`
+set, priority of pods with no `PriorityClassName` will be zero.
+
+`description` is an arbitrary string. It is meant to tell users of the cluster
+when they should use this PriorityClass.
+
+
+**Note 1:** If you upgrade your existing cluster and enable this feature, the priority
+of your existing pods will be considered to be zero.
+
+**Note 2:** Addition of a PriorityClass with `globalDefault` set to true does not
+change priority of existing pods. The value of such PriorityClass will be used only
+for pods created after the PriorityClass is added.
+
+#### Example PriorityClass
+```yaml
+apiVersion: v1
+kind: PriorityClass
+metadata:
+  name: high-priority
+value: 1000000
+globalDefault: false
+description: "This priority class should be used for XYZ service pods only."
+```
+
+## Pod Priority
+Once you have one or more PriorityClasses, you can create pods which specify one
+of those PriorityClass names in their spec.
+The following YAML is an example of a pod configuration that uses the PriorityClass
+created above. Priority admission controller checks the spec and resolves the
+priority of the pod to 1,000,000.
```

Comment:
do "1000000" instead of "1,000,000" because in a lot of places outside the US "," is the same as a US "." in numbers.